typeql/modules/ROOT/pages/values/duration.adoc

```diff
@@ -87,6 +87,34 @@ When used with a value of type datetime-tz, arithmetic operations with duration
 2024-03-30T12:00:00 Europe/London + PT24H == 2024-03-31T13:00:00 
 ----
 
+If adding months and days results in an ambiguous datetime in the time zone, the earlier datetime is used.
+
+[,typeql]
+----
+# London DST change occurred on 2024-10-27 02:00:00 BST
+# I.e., on 2024-10-27, the time span between 01:00:00 and 02:00:00 occurs twice
+2024-10-26T01:30:00 Europe/London + P1D == 2024-10-27T01:30:00 BST
+
+# Between these two local datetimes, the earliest is BST
+2024-10-26T01:30:00 Europe/London + PT24H == 2024-10-27T01:30:00 BST
+2024-10-26T01:30:00 Europe/London + PT25H == 2024-10-27T01:30:00 GMT
+----
```

Comment:
I struggled a little bit to follow why it was `Europe/London` on the left and `BST` or `GMT` on the right, but I get it now.

Reply:
I tried `2024-10-27T01:30:00 Europe/London(BST)` on the right, but unfortunately that was long enough to require scrolling and hid the actual timezone :/